test/units/module_utils/hwc/test_dict_comparison.py

```diff
@@ -0,0 +1,188 @@
+# -*- coding: utf-8 -*-
+# 2018.07.26 --- use DictComparison instead of GcpRequest
+#
+# (c) 2016, Tom Melendez <tom@supertom.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+import os
+import sys
+
+from units.compat import unittest
+from ansible.module_utils.hwc_utils import DictComparison
+
+
+class HwcDictComparisonTestCase(unittest.TestCase):
+    def test_simple_no_difference(self):
+        value1 = {
+            'foo': 'bar',
+            'test': 'original'
+        }
+        d = DictComparison(value1)
+        self.assertTrue(d == d)
```

Comment:
To avoid linter errors, change this to
```suggestion
        d_ = d
        self.assertTrue(d == d_)
```
and hope that the linter doesn't get too clever ;-)

Reply:
done